source/mentoring-service/source/agenda.md

```diff
@@ -55,13 +55,20 @@ In this activity, the most participating, interactive and experienced members we
 
 ## DevOps Public Session
 
-Members will give devops lectures in Arabic and English languages and it will be presented to the public to contribute to spreading knowledge and community awareness
+Members will give devops lectures in Arabic and English languages and it will be presented to the public to contribute to spreading knowledge and community awareness .
```

Comment:
@naja-ahmed Please when you use the  "." there is no space before it, the same for the ","